functional/keylime-policy-commands/test.sh

```diff
@@ -216,8 +222,8 @@ rlJournalStart
     # Sign runtime policies.
 
     rlPhaseStartTest "Sign runtime policy with the ECDSA DSSE backend"
-        mkdir -p sign-ecdsa
-        pushd sign-ecdsa || exit
+        rlRun "mkdir -p sign-ecdsa"
+        pushd sign-ecdsa || rlFail "Failed to create sign-ecdsa directory"
```

Comment:
Won't hurt but given the `rlRun` around `mkdir` above the former command would fail already. `pushd` alone could fail in case the user is not able to enter a previously created dir, which I think could be due to permissions etc. Either way, you could use also `rlRun pushd` and avoid explicit `rlFail` call. Just FYI, no need to change it.

Reply:
I think it would be cleaner with all running using `rlRun` and no explicit failure. I'll make the changes.

Reply:
Done